pyseer/similarity.py

```diff
@@ -104,8 +104,9 @@ def main():
                             block_size)
     while not eof:
         variants, variant_mat, eof = next(v_iter)
-        if G.shape[1] > block_size:
```

Comment:
This lead to this part of the code to never be invoked, the subsequent concatenate call also would not have worked